include/picongpu/param/radiation.param

```diff
@@ -43,6 +43,8 @@
 #include "picongpu/particles/manipulators/manipulators.def"
 
 namespace picongpu
+{
+namespace radiation
 {
     namespace rad_linear_frequencies
```

Comment:
Now when it is inside `picongpu::radiation`, the `rad` prefix is probably excessive (not only in this line).